Makefile

```diff
@@ -71,7 +82,7 @@ lint: $(INSTALL_STAMP) ## Run all linters
 	$(POETRY) run isort --check-only .
 	$(POETRY) run black --check . --diff
 	$(POETRY) run flake8 .
-	$(POETRY) run pylint  --recursive yes .
+	$(POETRY) run pylint  --recursive yes . --ignore=build
```

Comment:
Can we move this to `[tool.pylint.MASTER]` in pyproject.toml? there is already an exclusion list there.